uci-pharmsci/README.md

```diff
@@ -9,3 +9,29 @@ This will ultimately contain syllabus information, lecture notes and slides, req
 - `syllabus.md`: Syllabus for PharmSci 175/275
 - `175`: Undergraduate course (PharmSci 175)-specific materials (e.g. assignments)
 - `275`: Graduate course (PharmSci 275)-specific materials (e.g. assignments)
+
+## Getting Started
+- **[How to use Jupyter Notebook](http://jupyter-notebook-beginner-guide.readthedocs.io/en/latest/execute.html)**
+- **[Setup and Installation](getting-started.md)**
+
+## Requirements
+- [Anaconda Python](https://www.continuum.io/downloads): Anaconda is the leading open data science platform powered by Python.
+- [VMD](http://www.ks.uiuc.edu/Development/Download/download.cgi?PackageName=VMD): VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting.
+  - Recommended Tutorials:
+    - [Introduction](http://www.ks.uiuc.edu/Training/Tutorials/vmd/tutorial-html/node1.html)
+    - [Working with a Single Molecule](http://www.ks.uiuc.edu/Training/Tutorials/vmd/tutorial-html/node2.html)
+    - [Trajectories and Movie Making](http://www.ks.uiuc.edu/Training/Tutorials/vmd/tutorial-html/node3.html)
+- [OpenEye-toolkits](https://www.eyesopen.com/toolkit-development)
+  - [Documentation](https://docs.eyesopen.com/toolkits/python/index.html)
+- [XQuartz](https://www.xquartz.org/) - X11 for MacOS
+
+- **Optionals:**
+  - [Atom](https://atom.io/): Github's official text editor with so many plugins.
+  - [VIDA](https://www.eyesopen.com/vida): Molecular viewer for OEB files.
+
+#### Cheatsheets and dotfiles
+Under the **uci-pharmasci/docs/** directory, I've included some useful command cheat sheets for [BASH](https://github.com/nathanmlim/blues-apps/tree/master/docs/bash_cheatsheet.jpg) and [vi](https://github.com/nathanmlim/blues-apps/tree/master/docs/vi_cheatsheet.pdf).
+
+We also have a [Github](https://www.evernote.com/shard/s26/sh/ae73a67b-4d7a-4e97-a896-cef5473db895/178762935c73b559) cheat sheet/tutorial on our group Evernote. I also recommend checking out the [CodeAcademy Github](https://www.codecademy.com/learn/learn-git) tutorial.
+
+I've included some of my _dotfiles_ which may come in handy. Copy these to your HOME directory in GreenPlanet (and your local machine if needed).
```

Comment:
Remove greenplanet here?

Add yourself to acknowledgments in main README.md? 

Explain what a dotfile is or link to explanation?